packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -878,3 +877,4 @@ if (conf.get('ecosystem_anon_id.keys_file')) {
 
 // validate the configuration based on the above specification
 // conf.validate(options);
+module.exports.conf = conf;
```

Comment:
No need for this.

You can find examples of how the configs are passed into routes in `server/lib/routes.js`.